### PR TITLE
FLUT-6089 - [Others] UG improvements and missing properties added

### PR DESCRIPTION
**Pull request by PavithraNandhiniRamachandran**
## Description ##

AnchorRangeToVisiblePoint property added in cartesian chart UG.
added the tab and end tab in all code snippets.
added the track opacity property in radial bar chart

### CODE REVIEW

**Comment by PavithraNandhiniRamachandran** [comment]
rebuild

**Comment by PavithraNandhiniRamachandran** [comment]
rebuild

File: Flutter/cartesian-charts/zoom-pan.md [hunk 1]
```diff
@@ -329,6 +329,60 @@ If zoom mode is set to [`zoomMode.x`](https://pub.dev/documentation/syncfusion_f
...
+    Widget build(BuildContext context) {
+      return Scaffold(
+        body: SafeArea(
+          child: Center(
+            child: Container(
+              child: SfCartesianChart(
+                zoomPanBehavior: _zoomPanBehavior,
+                 primaryXAxis: DateTimeAxis(),
+                 primaryYAxis: NumericAxis(
+                          anchorRangeToVisiblePoints: false,),
```
Comment (dharanidharandharmasivam) [inline Flutter/cartesian-charts/zoom-pan.md:367]: remove extra comma

File: Flutter/cartesian-charts/zoom-pan.md [hunk 2]
```diff
@@ -329,6 +329,60 @@ If zoom mode is set to [`zoomMode.x`](https://pub.dev/documentation/syncfusion_f
...
+                 series: <ChartSeries<ChartData, DateTime>>[
+                      AreaSeries<ChartData, DateTime>(
+                          dataSource: chartData,
+                          xValueMapper: (ChartData sales, _) => sales.x,
+                          yValueMapper: (ChartData sales, _) => sales.y,s
```
Comment (dharanidharandharmasivam) [inline Flutter/cartesian-charts/zoom-pan.md:372]: why there is s character at last

**Comment by dharanidharandharmasivam** [comment]
@PavithraNandhiniRamachandran ,

1. This is not customer bug, so provide proper PR title
2. The gif image is not proper, while zooming and panning, y-value needs to be smaller for example it can be less than or equal to 75

**Review by dharanidharandharmasivam — CHANGES_REQUESTED** [review]

File: Flutter/cartesian-charts/zoom-pan.md (hunk 1 above)
Comment (PavithraNandhiniRamachandran) [reply Flutter/cartesian-charts/zoom-pan.md:367]: removed the extra commands

File: Flutter/cartesian-charts/zoom-pan.md (hunk 2 above)
Comment (PavithraNandhiniRamachandran) [reply Flutter/cartesian-charts/zoom-pan.md:372]: removed the s

**Comment by PavithraNandhiniRamachandran** [comment]
rebuild

File: Flutter/cartesian-charts/annotations.md [hunk 3]
```diff
@@ -52,6 +54,7 @@ The [`x`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/
 To position the annotation based on axis, set the [`x`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/CartesianChartAnnotation/x.html) and [`y`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/CartesianChartAnnotation/y.html) properties based on axis range values, and set the [`coordinateUnit`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/CartesianChartAnnotation/coordinateUnit.html) value as [`CoordinateUnit.point`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/CartesianChartAnnotation/coordinateUnit.html).
 
 
+{% tabs %}
```
Comment (dharanidharandharmasivam) [inline Flutter/cartesian-charts/annotations.md:57]: where is the closing tag? I couldn't find this in most of the places

{% endtabs %}

File: Flutter/cartesian-charts/zoom-pan.md [hunk 4]
```diff
@@ -327,6 +335,11 @@ If zoom mode is set to [`zoomMode.x`](https://pub.dev/documentation/syncfusion_f
 
 Also refer [`zooming`](./callbacks#onzooming), [`zoom start`](./callbacks#onzoomstart) and [`zoom end`](./callbacks#onzoomend) events for customizing the zooming further.
 
+### Calculate range based on visible points
+
+[`anchorRangeToVisiblePoints`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/ChartAxis/anchorRangeToVisiblePoints.html) property can be set to `false` the value axis range will be calculated based on all the data points in the chart irrespective of visible points. For further reference, check this [AutoRangeCalculation](./axis-customization.md) section.
+
```
Comment (dharanidharandharmasivam) [inline Flutter/cartesian-charts/zoom-pan.md:341]: why the gif image for anchorRangeToVisiblePoints exists? already conveyed to remove this

**Comment by dharanidharandharmasivam** [comment]
Hi @PavithraNandhiniRamachandran ,

Have proper PR title and description

Thanks,
Dharani.

File: Flutter/cartesian-charts/annotations.md (hunk 3 above)
Comment (PavithraNandhiniRamachandran) [reply Flutter/cartesian-charts/annotations.md:57]: have added the end tag

File: Flutter/cartesian-charts/zoom-pan.md (hunk 4 above)
Comment (PavithraNandhiniRamachandran) [reply Flutter/cartesian-charts/zoom-pan.md:349]: have removed the gif image

File: Flutter/cartesian-charts/annotations.md (hunk 3 above)
Comment (PavithraNandhiniRamachandran) [reply Flutter/cartesian-charts/annotations.md:57]: added close tag

File: Flutter/cartesian-charts/zoom-pan.md (hunk 4 above)
Comment (PavithraNandhiniRamachandran) [reply Flutter/cartesian-charts/zoom-pan.md:349]: yes have removed

**Comment by PavithraNandhiniRamachandran** [comment]
rebuild

File: Flutter/circular-charts/chart-types/radial-bar-chart.md [hunk 5]
```diff
@@ -155,9 +162,56 @@ The [`cornerStyle`](https://pub.dev/documentation/syncfusion_flutter_charts/late
     }
 
 {% endhighlight %}
+{% endtabs %}
 
 ![Radial bar corner style](circular-chart-types-images/radialbar_roundCorner.jpg)
 
+## Track opacity
```
Comment (dharanidharandharmasivam) [inline Flutter/circular-charts/chart-types/radial-bar-chart.md:169]: Remove the heading and content and use the below content. Also move this after `Rounded corners` section and need to remove the old content as in below screenshot

image


## Track customization

The radial bar series renders above the track area. The following properties are used to customize the appearance.

* [`trackColor`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/RadialBarSeries/trackColor.html) - changes the color of the track area.
* [`trackBorderColor`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/RadialBarSeries/trackBorderColor.html) - changes the stroke color of the track area.
* [`trackBorderWidth`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/RadialBarSeries/trackBorderWidth.html) - changes the stroke width of the track area.
* [`trackOpacity`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/RadialBarSeries/trackOpacity.html) - specifies the opacity of the track area.
* [`useSeriesColor`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/RadialBarSeries/useSeriesColor.html) - uses the point color for filling the track area.

**Review by dharanidharandharmasivam — CHANGES_REQUESTED** [review]

File: Flutter/cartesian-charts/callbacks.md [hunk 6]
```diff
@@ -24,7 +24,8 @@ Triggers when the tooltip is rendering. Here, you can customize the text, header
 * [`pointIndex`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/TooltipArgs/pointIndex.html) - specifies the current point index.
 * [`viewportPointIndex`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/TooltipArgs/viewportPointIndex.html) - specifies the viewport index value of the tooltip.
 
- {% highlight dart %}
+ {% tabs %}
```
[the remark this answers is not in the document]
Comment (PavithraNandhiniRamachandran) [reply Flutter/cartesian-charts/callbacks.md:27]: aligned properly

File: Flutter/circular-charts/chart-types/radial-bar-chart.md (hunk 5 above)
Comment (PavithraNandhiniRamachandran) [reply Flutter/circular-charts/chart-types/radial-bar-chart.md:169]: added the above content

File: Flutter/circular-charts/chart-types/radial-bar-chart.md [hunk 7]
```diff
@@ -155,9 +162,56 @@ The [`cornerStyle`](https://pub.dev/documentation/syncfusion_flutter_charts/late
...
+                    )
+                )
+            )
+        );
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+![Track opacity](circular-chart-types-images/trackopacity.png)
```
[the remark this answers is not in the document]
Comment (PavithraNandhiniRamachandran) [reply Flutter/circular-charts/chart-types/radial-bar-chart.md:213]: changed

File: Flutter/cartesian-charts/chart-types/stacked-column-chart.md [hunk 8]
```diff
@@ -134,14 +137,16 @@ You can group and stack the similar stacked series types using the [`groupName`]
...
 
 ![Stacked column grouping](cartesian-chart-types-images/stacked_column_grouping.jpg)
 
 ## Display cumulative values
 
 You can show the cumulative data label values using the [`showCumulativeValues`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/DataLabelSettings/showCumulativeValues.html) property. If the series are grouped using [`groupName`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/StackedColumnSeries/groupName.html), then cumulative values will be shown based on grouping.
 
-{% highlight dart hl_lines="13 23 33 43" %} 
+{% tabs %}
+{% highlight dart hl_lines="13 23 33 43 %}
```
[the remark this answers is not in the document]
Comment (PavithraNandhiniRamachandran) [reply Flutter/cartesian-charts/chart-types/stacked-column-chart.md]: In local it was correct i will check the issue 
image

**Comment by PavithraNandhiniRamachandran** [comment]
rebuild

**Review by dharanidharandharmasivam — APPROVED** [review]

**Pull request merged**